packages/sui-ssr/server/dynamicRendering/index.js

```diff
@@ -2,17 +2,16 @@ import path from 'path'
 import fs from 'fs'
 import seoBotDetect from './seoBotDetect'
 
-const {DYNAMIC_RENDERING = false} = process.env
 const INDEX_HTML_PATH = path.join(process.cwd(), 'public', 'index.html')
 
-export default function dinamycRendering(fallback) {
+export default function dinamycRendering(fallback, dynamicsURLS = []) {
```

Comment:
I know it was already wrong, but it's spelled `dynamic`. Maybe it's a good time to change the function name.

Reply:
👍